tb_plugin/torch_tb_profiler/profiler/trace.py

```diff
@@ -103,6 +103,29 @@ def __init__(self, type, data):
 
         self.device_type = dtype
 
+    @property
+    def addr(self):
+        return self.args.get("Addr")
+
+    @property
+    def bytes(self):
+        return self.args.get("Bytes", 0)
+
+    @property
+    def total_allocated(self):
+        # TODO: Allocated Bytes to be renamed to Total Allocated
+        old = self.args.get("Allocated Bytes")
+        if old is not None:
+            return old
+        return self.args.get("Total Allocated", float("nan"))
```

Comment:
why not None?

Reply:
Because it is a float. Just like how pandas treating missing value mentioned in https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html
> The choice of using NaN internally to denote missing data was largely for simplicity and performance reasons. Starting from pandas 1.0, some optional data types start experimenting with a native NA scalar using a mask-based approach. See here for more.
No preference here, they will be filtered out latter before returning to frontend.